.github/workflows/mock-test-python-ci.yml

```diff
@@ -0,0 +1,21 @@
+name: test-python-ci
+
+on:
+  pull_request:
+    paths:
+      - "**"
+      - "!parsec/**"
+      - "!tests/**"
+      - "!src/**.rs"
```

Comment:
This is not matching the regex in `test-python-ci.yml` (it is `src/**` there)

Reply:
While you're fixing that, can you also add a comment in each file `# Must be kept in sync with xxx/yyy.yml !`

Reply:
- for the matching pattern (not regex), done in c766a41
- for the comment, done in bc8a5f5